modules/plugins/utility/new-file-template/new-file-template.nix

```diff
@@ -0,0 +1,53 @@
+{
+  lib,
+  ...
+}: let
+  inherit (lib.options) mkOption;
+  inherit (lib.types) attrsOf bool listOf str;
+  inherit (lib.nvim.types) mkPluginSetupOption;
+in {
+  options.vim.utility.new-file-template = {
+    enable = mkOption {
+      type = bool;
+      default = false;
+      description = ''
+        new-file-template.nvim: Automatically insert a template on new files in neovim.
+        To add custom templates add a directory containing `lua/templates/*.lua` to `vim.additionalRuntimePaths`.
```

Comment:
Wrap this line in 
```
::: {.note}

:::
```

to mark this line as a note annotation in the documentation.

Also, limit characters per line to 80 and use marked links for the URL.